scripts/ts-generate-definition.js

```diff
@@ -39,8 +39,12 @@ fs.readFile(clientFile, 'utf8', (err, data) => {
           return block;
         }
         const nullableBlock = block.split('\n')
-          .map(line => line.replace('?: ', ': null | '))
-          .join('\n');
+          .map(line => {
+            return line.replace(
+              /^ {2}([a-zA-Z0-9_$]+)\?: ([a-zA-Z0-9_$<>]+)/,
```

Comment:
The idea is to find the lines which have, in sequence:

* `^` a beginning of line
* ` {2}` two spaces
* `([a-zA-Z0-9_$]+)` a field identifier (note: first group `$1`)
* `\?: ` the characters `?`, `:` and space
* `([a-zA-Z0-9_$<>]+)` a type identifier (note: second group `$2`)

I'm using a rather strict regex to avoid affecting lines we don't want to affect:
* line 60 of index.d.ts: `  createRequest(uri: string, method?: string, body?: Object): T;`
* line 93-95 of index.d.ts. Line 92 is `  overrides: {`. The following line have an indent of 4 and I think they would be undefined rather than nullable.
* line 133 of index.d.ts: `  follower_ids?: Array<ID>;`

However, if we decide to use block annotation, I might simplify this regex. Maybe even revert it back to `line.replace('?: ', ': null | ')`.